<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExQSlider</class>
 <widget class="QWidget" name="ExQSlider">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExQSlider</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>701</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="1">
     <widget class="QLabel" name="labRGBVal">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Red</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QSlider" name="sliderGreen">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSlider" name="sliderRed">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Alpha</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QSlider" name="sliderAlpha">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Green</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Blue</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="8">
     <widget class="QTextEdit" name="textColor"/>
    </item>
    <item row="5" column="0">
     <widget class="QSlider" name="sliderBlue">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
